templates/pages/page.html.twig

```diff
@@ -430,37 +430,33 @@
                                 {# Don't display it #}
                             {% else %}
                                 {% if page.content.full_width_content %}
-                                    <div class="apollo-island apollo-island--emphasised">
-                                        <div class="govuk-width-container">
-                                            <div class="govuk-grid-row">
+                                    {% if page.content.option_cards.value|default(null) %}
+                                    {# display if option cards exists and is true #}
+                                        <div class="apollo-island apollo-island--emphasised">
+                                            <div class="govuk-width-container">
+                                                <div class="govuk-grid-row">
+                                                    <div class="govuk-grid-column-one-half">
+                                                        <h2 class="visuallyhidden">You might also be interested in:</h2>
```

Comment:
I know this is in existing code, but just wondering, is this for accessibility?

Reply:
Yes, I believe this is for screen readers.